README.md

```diff
@@ -152,6 +152,70 @@ A `jump box` pod is created so that you can execute commands `in the cluster`
   - Click `Execute`
   - This will display the `histogram` that Grafana uses for the charts
 
+## Expose Grafana Service
+
+> Goal: Make the Grafana dashboard accessible via a Codespaces forwarded port
+
+Forward the soon-to-be mapped port on Codespaces
+
+- Open `.devcontainer/devcontainer.json`
+- Add port 32000 to `forwardPorts`
+- Add the port label `"32000": { "label": "Grafana" }` to `portsAttributes`
+- Select green `Codespaces` on the bottom left-hand side of VSCode
+- Select `Rebuild Container` from the dropdown at the top of VSCode
+- Verify the addition of `Grafana (32000)` in the `Ports` tab of the Terminal view
+
+Set the NodePort on Grafana's deployed service
+
+- Open `deploy/grafana/deployment.yaml`
+- In the configs for `kind: service`, find the `targetPort`
+- Add `nodePort: 32000` below `targetPort: 3000`. INDENTATION MATTERS!
+
+```yaml
+  ports:
+    - port: 3000
+      targetPort: 3000
+      nodePort: 32000
+```
+
+Map the NodePort to a local port
```

Comment:
I wouldn't actually delete the Grafana ports - I would just document the three places to add the port mapping.

Reply:
Removed deletion of Grafana ports